cmd/mockery.go

```diff
@@ -87,20 +87,6 @@ func NewRootCmd() *cobra.Command {
 	return cmd
 }
 
-type stackTracer interface {
```

Comment:
We could alternatively choose to preserve stack traces by implementing a thin error wrapper for internal use: https://github.com/vektra/mockery/pull/638/files#diff-8c40867e9f5267a12ce0915f1f39d2d0842f0acae399ee8674d4d621e1c01871R1